Userland/Services/SpiceAgent/SpiceAgent.cpp

```diff
@@ -121,16 +121,19 @@ ErrorOr<void> SpiceAgent::on_clipboard_message(ClipboardMessage& message)
 
 ErrorOr<ByteBuffer> SpiceAgent::read_message_buffer()
 {
-    // TODO(Caoimhe): `ChunkHeader` class?
```

Comment:
since you actually added the chunk header in this PR, can we yeet the TODO comments from the earlier commits?